tests/test_schema.py

```diff
@@ -1,9 +1,16 @@
 import unittest
 import copy
 from pydantic import parse_obj_as, ValidationError
+from typing_extensions import Any
 
 from bpx import BPX
 
+from pydantic import TypeAdapter
+
+# bpxobject = BPX()
+
+adapter = TypeAdapter(dict[str, dict[str, float | str | dict[str, Any]]])
```

Comment:
Here @agriyakhetarpal & I were stuck trying to configure  bpx object to be parsed in `TypeAdapter`.